lib/sisimai/bite/email/x3.rb

```diff
@@ -0,0 +1,141 @@
+module Sisimai::Bite::Email
+  module X3
+    # Sisimai::Bite::Email::X3 parses a bounce email which created by Unknown
+    # MTA #3. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/X3.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/\AMail Delivery System/,
+        :subject => %r/\ADelivery status notification/,
+      }
+      Re1 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->